LibOS/shim/src/bookkeep/shim_vma.c

```diff
@@ -630,6 +652,16 @@ int bkeep_munmap (void * addr, uint64_t length, int flags)
     return ret;
 }
 
+/*
+ * Update bookkeeping for mprotect(). "prev" must be passed into the function
+ * as the immediate precedent vma of the protecting address, or NULL if no
```

Comment:
`protecting` -> `protected`
plus the same issues as with the `__bkeep_munmap` (see the comment under it)

Reply:
Rewritten.